rustfmt-core/rustfmt-lib/src/attr.rs

```diff
@@ -1,6 +1,6 @@
 //! Format attributes and meta items.
 
-use rustc_span::{BytePos, DUMMY_SP, Span, symbol::sym};
+use rustc_span::{symbol::sym, BytePos, Span, DUMMY_SP};
```

Comment:
Sorry, I had a feeling I was adding some misformatted things when bumping the rustc-ap crates 😞. Glad to have the tests running again!